packages/sdks/src/functions/get-content/generate-content-url.ts

```diff
@@ -15,6 +15,7 @@ export const generateContentUrl = (options: GetContentOptions): URL => {
     model,
     apiKey,
     includeRefs = true,
+    enrich,
```

Comment:
Two things:

- I can see here that `generateContentUrl` sets default values for `noTraverse` and `includeRefs`. We end up providing those two along with `enrich`. Do we still want to do that? Would that cause any issues? Or do we want to not provide those defaults if `enrich` is provided for us?
- shouldn't `enrich` have a default of `true`, since `includeRefs` has a default of `true`?